packages/web-api/src/response/ConversationsHistoryResponse.ts

```diff
@@ -82,6 +82,7 @@ export interface Attachment {
   thumb_url?:             string;
   thumb_width?:           number;
   thumb_height?:          number;
+  video_url?:             string;
```

Comment:
These unrelated changes are the ones detected by recent. integration tests in the Java SDK project